awx/main/models/ha.py

```diff
@@ -390,7 +390,7 @@ def find_largest_idle_instance(instances, capacity_type='execution'):
         for i in instances:
             if i.node_type not in (capacity_type, 'hybrid'):
                 continue
-            if i.jobs_running == 0:
+            if (hasattr(i, 'jobs_running') and i.jobs_running == 0) or i.remaining_capacity == i.capacity:
```

Comment:
I figure `jobs_running == 0` is equivalent to `remaining_capacity == total_capacity` and both indicate an "idle" instance